saleor/graphql/discount/schema.py

```diff
@@ -2,27 +2,31 @@
 from graphql_jwt.decorators import permission_required
 
 from ..core.fields import PrefetchingConnectionField
-from ..descriptions import DESCRIPTIONS
+# FIXME: Types are imported before mutations on purpose. Otherwise these types
```

Comment:
What if we imported these at the beginning of `mutations.py`?

Reply:
We could do that as well. Actually, I think there is no much difference - either here or there `isort` will complain about the wrong order of imports. I have no idea why this happens, I spent one hour trying to debug it but found nothing.